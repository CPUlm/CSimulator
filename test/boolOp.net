INPUT a,b
OUTPUT c,d,e,f
VAR
  a , b, c,d,e,f
IN
c = NOT a
d = AND a b
e = NAND a b
f = OR a b
