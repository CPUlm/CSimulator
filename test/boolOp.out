Step 1: 
=> c = 1
=> d = 0
=> e = 1
=> f = 0
Step 2: 
=> c = 0
=> d = 0
=> e = 1
=> f = 1
Step 3: 
=> c = 1
=> d = 0
=> e = 1
=> f = 1
Step 4: 
=> c = 0
=> d = 1
=> e = 0
=> f = 1
